.chloggen/stanza-operator-move-struct.yaml

```diff
@@ -0,0 +1,9 @@
+change_type: enhancement
```

Comment:
Let's skip the changelog entry on this one, since it has no user facing changes.

Reply:
ok